### PR TITLE
fix port preference update

### PR DESCRIPTION
**Pull request by rbioteau**

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/preferences/BonitaUserXpPreferencePage.java [hunk 1]
```diff
@@ -157,7 +157,7 @@ public void run(final IProgressMonitor monitor) throws InvocationTargetException
     public void propertyChange(final PropertyChangeEvent event) {
         if (event.getSource().equals(port)) {
             if (port.isValid()) {
-                newPort = Integer.valueOf((String) event.getNewValue());
+                newPort = port.getIntValue();
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/preferences/BonitaUserXpPreferencePage.java]: we don't use anymore the "newValue" sent by the event? while reading it seems to me that we are reusing the old port value

Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/preferences/BonitaUserXpPreferencePage.java]: we use it in performOk

Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/preferences/BonitaUserXpPreferencePage.java]: ok

**Comment by apupier** [comment]
OK to merge when CI is green

**Pull request merged**